tests/static/commtrack/balance-block.xml

```diff
@@ -14,7 +14,8 @@
 			<instance id="ledger" src="jr://instance/ledgerdb" />
 			<instance id="products" src="jr://fixture/commtrack:products" />
 			<bind nodeset="/data/stock_amount" type="xsd:int" />
-			<bind nodeset="/data/balance[@type='bal-0']/entry/@quantity" calculate="/data/stock_amount" relevant="/data/stock_amount != 0" />
+			<bind nodeset="/data/balance[@type='bal-0']" relevant="/data/stock_amount != 0" />
+			<bind nodeset="/data/balance[@type='bal-0']/entry/@quantity" calculate="/data/stock_amount" />
```

Comment:
:+1: this looks right to me. thanks for quick response and sorry i didn't catch this the first go around.